connexion/exceptions.py

```diff
@@ -95,3 +96,20 @@ def __init__(self, token_scopes, required_scopes, title=None, **kwargs):
         self.missing_scopes = required_scopes - token_scopes
 
         super(OAuthScopeProblem, self).__init__(title=title, **kwargs)
+
+
+class ExtraParameterProblem(ProblemException, BadRequest):
+    def __init__(self, formdata_parameters, query_parameters, title=None, description=None, **kwargs):
```

Comment:
So with this idea we would have a handle of this exception that could render something like:

```json
{
   "type": "https://example.net/validation-error",
   "title": "Your request has extra parameters.",
   "extra-query-params": ["name", "age"],
   "extra-formdata-params": ["reason", "color"]
}
```